sdk/src/test/java/com/mapbox/maps/renderer/MapboxRendererTest.kt

```diff
@@ -47,14 +44,53 @@ internal abstract class MapboxRendererTest {
   @Test
   fun scheduleRepaintTest() {
     mapboxRenderer.scheduleRepaint()
-    verify { renderThread.requestRender() }
+    verify {
+      renderThread.queueRenderEvent(
+        RenderEvent(
+          null,
+          true,
+          EventType.SDK
+        )
+      )
+    }
   }
 
   @Test
   fun scheduleTaskTest() {
     val task = mockk<Task>(relaxUnitFun = true)
+    every { renderThread.nativeRenderDestroyCallChain } returns false
+    var actualNeedRender: Boolean? = null
+    var actualEventType: EventType? = null
+    every { renderThread.queueRenderEvent(any()) } answers {
+      actualNeedRender = firstArg<RenderEvent>().needRender
+      actualEventType = firstArg<RenderEvent>().eventType
+    }
     mapboxRenderer.scheduleTask(task)
-    verify { renderThread.queueEvent(any()) }
+    verify {
+      // no idea how to verify Task got transformed in correct lambda so checking call + desired properties
+      renderThread.queueRenderEvent(any())
```

Comment:
Could capture runnable and call it, then check if task will be called

Reply:
e.g.
```
  var captureTo: RenderEvent
  renderThread.queueRenderEvent(capture(captureTo))
  capture.runnable.run()
```

Reply:
Well capture works https://github.com/mapbox/mapbox-maps-android/pull/1068/files#diff-57f2dab0d3da1120c8042e3b8e54ae41eb8ef84734ba6f77e51dc373ef33ac59R268 right? Wouldn't the same work here?